reconcile/aus/models.py

```diff
@@ -16,3 +21,93 @@ class ClusterUpgradeSpec(BaseModel):
 class OrganizationUpgradeSpec(BaseModel):
     org: AUSOCMOrganization
     specs: list[ClusterUpgradeSpec]
+
+
+class ConfiguredUpgradePolicyConditions(BaseModel):
```

Comment:
I don't understand how this class is different or brings value compared to `ClusterUpgradePolicyConditionsV1` (which is included in `ClusterUpgradePolicy`) in `reconcile.gql_definitions.fragments.upgrade_policy`.

Same for `ConfiguredUpgradePolicy` below

Reply:
For `ConfiguredUpgradePolicyConditions`, the sector attribute will have a reference to the OCM sector. 
For the other classes, i.e. `ConfiguredClusterUpgradePolicy` it has additional attributes like 

- current_version
- channel
- available_upgrades

Reply:
Ack, thanks